content/_data/changelogs/weekly.yml

```diff
@@ -83,6 +83,13 @@
       message: Remove invalid translations in Slovene
       issue: 41756
       pull: 2767
+- version: "2.50"
+  date: 2017-03-11
+  changes:
+    - type: rfe
+      message: Expose the noun for an item when behaving as a task via <tt>AlternativeUiTextProvider</tt>.
```

Comment:
We typically put a "Developer" or "API" prefix for things that don't make a lot of sense to users, can be done on merge or in preparation of the release.